src/extensions/mvs/helpers/selections.ts

```diff
@@ -280,7 +281,13 @@ export function rowToExpression(row: MVSAnnotationRow): Expression {
     }
 
     const residueTests: Expression[] = [];
-    if (isDefined(row.label_seq_id)) residueTests.push(eq([macromolecular.label_seq_id(), row.label_seq_id]));
+    if (isDefined(row.label_seq_id)) {
+        if (row.element_granularity === 'coarse') {
+            residueTests.push(ihm.hasSeqId({ 0: row.label_seq_id }));
+        } else {
+            residueTests.push(eq([macromolecular.label_seq_id(), row.label_seq_id]));
+        }
+    }
```

Comment:
Does this mean that if `label_seq_id` is not present, it will select from both atomic and coarse parts of the structure?

Reply:
Actually, we can completely remove `element_granularity` in this case and just use the `ihm.hasSeqId`.

Reply:
Hmmm, if we can remove `element_granularity`, let's do it. One thing less for the user to worry about.